### DISCUSSION TITLE
Weird autoplay mode usernames in different rulesets

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#15401] [Q&A] [answered]
![ScreenShot_20211102004922](https://user-images.githubusercontent.com/86934170/139747317-f334b38c-490e-4ab5-b49a-4adc181090bb.png)
![ScreenShot_20211102004959](https://user-images.githubusercontent.com/86934170/139747319-b8196bf3-fc14-436a-bf71-c0344045b9b3.png)
![ScreenShot_20211102005044](https://user-images.githubusercontent.com/86934170/139747323-28041d12-b920-42c5-8bf3-00bcba5614a4.png)
![ScreenShot_20211102005125](https://user-images.githubusercontent.com/86934170/139747324-1667af02-6246-42aa-af0e-1764dfdfbc52.png)

For osu!/taiko/catch/mania relatively

**Comment by bdach** [comment, marked as answer]
this is intentional.

**Reply by peppy** [reply]
I think the trailing exclamation marks on some of them should be removed.

Also they will likely make more sense once they have proper (placeholder) avatars.

**Reply by smoogipoo** [reply]
https://github.com/ppy/osu/pull/15407
